style.css

```diff
@@ -0,0 +1,58 @@
+* {
+  margin: 0;
+}
+
+body {
+  background-color: #eee;
+}
+
+#img-div {
+  background-color: white;
+  max-width: 100%;
+}
+
+#image {
+  height: auto;
+  max-width: 100%;
+  display: block;
+  margin: 0 auto !important;
+}
+
+#tribute-info {
+  width: 700px;
+  margin: 0 auto;
+}
+
+.blockquote {
+  width: 600px;
+  margin: 0 auto;
+}
+#main {
+  padding-top: 30px;
+  padding-bottom: 30px;
+  margin-bottom: 10px;
+}
+#title {
+  text-align: center;
+  font-size: 40px;
+  padding-bottom: 20px;
+}
+p {
+  text-align: center;
+  font-size: 26px;
+  padding-bottom: 20px;
+}
+ul {
+  line-height: 1.6;
+}
+li {
+  padding-bottom: 15px;
+}
+blockquote {
+  text-align: center;
+  font-size: 20px;
+  font-style: italic;
+}
+h3 {
+  text-align: center;
+}

```

Comment:
There should be an empty line before starting the next CSS rule.

Reply:
> There should be an empty line before starting the next CSS rule.

- [ ] Please, why do you think there should be an empty line? I am asking because there should be the WHY just as you did in your previous comment